Sources/ApolloCodegenLib/IR/IR+OperationBuilder.swift

```diff
@@ -12,7 +12,10 @@ extension IR {
 
     let rootEntity = Entity(
       rootTypePath: LinkedList(operationDefinition.rootType),
-      fieldPath: [.init(name: rootField.name, type: rootField.type)]
+      fieldPath: [.init(
+        name: "\(operationDefinition.generatedDefinitionName).Data",
```

Comment:
This is really hacky and a violation of separation of responsibilities. I'm working on a refactor that provides the information about the IR needed in order for the templates to format the name properly.

I'll be putting this PR back into draft.